pages/courses_page.py

```diff
@@ -98,3 +98,22 @@ def verify_resume_course_title(self):
         self.wait_for_element(course_title)
         resume_course_title = self.get_element_text(course_title)
         return resume_course_title
+
+    def verify_course_price_in_cart(self, full_course_name):
+        time.sleep(3)
+        self.explore_courses()
+        time.sleep(3)
```

Comment:
Why are these sleeps needed? They are already inside the methods that we are calling. Please remove them.